### PR TITLE
Incorporate native habitat data and info sources

### LINKED ISSUE #20: Add native habitat/range to each ambassador
e.g. Toaster Strudel -> Australia and New Guinea (forests, scrublands, and deserts)

### PR DESCRIPTION
**Pull request by itsjayway**
Resolves #20

### CODE REVIEW

**Comment by itsjayway** [comment]
running `npm install` added 

`      "license": "SEE LICENSE IN LICENSE.md",` entry to `package-lock.json`; I can add it to this PR if it would help

**Comment by MattIPv4** [comment]
You should be running `npm ci`, not `npm install`, which will maintain the current lockfile state.

**Comment by MattIPv4** [comment]
Could you run `npm version --no-git-tag-version minor` and push that up, so the version gets bumped with this change, please?

File: src/ambassadors/core.ts [hunk 1]
```diff
@@ -793,6 +909,10 @@ const ambassadors = {
       "Chips Ahoy was bred as a feeder rat for snakes, but she will now live out her full life at Alveus.",
     mission:
       "She is an ambassadors for how rodenticide use and outdoor cats affect all levels of wildlife.",
+    native: {
```
Comment (MattIPv4) [inline src/ambassadors/core.ts:859]: I would probably say that these rats (`f. domestica`) are worldwide are they are domesticated?

File: src/ambassadors/core.ts [hunk 2]
```diff
@@ -816,6 +936,10 @@ const ambassadors = {
       "Nilla Wafer was bred as a feeder rat for snakes, but she will now live out her full life at Alveus.",
     mission:
       "She is an ambassadors for how rodenticide use and outdoor cats affect all levels of wildlife.",
+    native: {
```
Comment (MattIPv4) [inline src/ambassadors/core.ts:886]: Ditto

File: src/ambassadors/core.ts (hunk 1 above)
Comment (MattIPv4) [reply src/ambassadors/core.ts:859]: https://en.wikipedia.org/wiki/Fancy_rat might also be a better wiki article for the domesticated rat?

Comment (itsjayway) [reply src/ambassadors/core.ts:859]: ah gotcha, my mistake. thank you for catching that

File: src/ambassadors/core.ts [hunk 3]
```diff
@@ -816,6 +936,11 @@ const ambassadors = {
...
+      text: "Worldwide",
+      source:
+        "https://en.wikipedia.org/wiki/Fancy_rathttps://en.wikipedia.org/wiki/Fancy_rat",
```
Comment (MattIPv4) [inline src/ambassadors/core.ts:942]:
```suggestion
        "https://en.wikipedia.org/wiki/Fancy_rat",
```

Comment (MattIPv4) [inline src/ambassadors/core.ts]:
```suggestion
      source: "https://en.wikipedia.org/wiki/Fancy_rat",
```

**Comment by pjeweb** [comment]
For the chickens. Would it be more correct to list them all as domesticated and not the country they were bred (USA/China)? Or we can make the field `origin` instead of (`native`). Imho we can also `null` native entries if they aren't wild species?!

**Comment by MattIPv4** [comment]
> For the chickens. Would it be more correct to list them all as domesticated and not the country they were bred (USA/China)? Or we can make the field `origin` instead of (`native`).

From an educational perspective, I think it's useful to list where they originally came from. I think we can add `(Domesticated)` afterwards though (the donkeys + winnie + some of the chickens already have this).

I think we should probably avoid the nomenclature of an `origin` prop here, as that might be misinterpreted to be the origin of the specific ambassador, rather than the species as a whole.

> Imho we can also `null` native entries if they aren't wild species?!

I think `null` leaves a lot of ambiguity, I think having an explicit bit of text that explains the situation each time is probably better. For example, it is much more useful to say to folks that Miley's species is bred in captivity than it is to just not list anything.

File: src/ambassadors/core.ts [hunk 4]
```diff
@@ -558,6 +634,10 @@ const ambassadors = {
       "Snork was part of an educational program at a zoo and was rehomed to Alveus.",
     mission:
       "She is an ambassador for the exploitation of wildlife in the fur trade.",
+    native: {
+      text: "Andes Mountains (South America)",
```
Comment (MattIPv4) [inline src/ambassadors/core.ts]: Should this be marked as domesticated?

File: src/ambassadors/core.ts [hunk 5]
```diff
@@ -580,6 +660,10 @@ const ambassadors = {
     story: "Rehomed from a local pet owner.",
     mission:
       "He is an ambassador for the exploitation of wildlife in the fur trade.",
+    native: {
+      text: "Andes Mountains (South America)",
```
Comment (MattIPv4) [inline src/ambassadors/core.ts]: Ditto?

File: src/ambassadors/core.ts [hunk 6]
```diff
@@ -793,6 +909,10 @@ const ambassadors = {
       "Chips Ahoy was bred as a feeder rat for snakes, but she will now live out her full life at Alveus.",
     mission:
       "She is an ambassadors for how rodenticide use and outdoor cats affect all levels of wildlife.",
+    native: {
+      text: "Worldwide",
```
Comment (MattIPv4) [inline src/ambassadors/core.ts]: Should this be flagged as domesticated?

File: src/ambassadors/core.ts [hunk 7]
```diff
@@ -816,6 +936,10 @@ const ambassadors = {
       "Nilla Wafer was bred as a feeder rat for snakes, but she will now live out her full life at Alveus.",
     mission:
       "She is an ambassadors for how rodenticide use and outdoor cats affect all levels of wildlife.",
+    native: {
+      text: "Worldwide",
```
Comment (MattIPv4) [inline src/ambassadors/core.ts]: Ditto?

File: src/ambassadors/core.ts [hunk 8]
```diff
@@ -937,6 +1077,10 @@ const ambassadors = {
       "Henrietta was rehomed from a local farm.\n\nHenrietta was retired from being an ambassador in March 2023 and now lives in a chicken coop at Maya's Tiny House with her rabbit and other chicken.",
     mission:
       "She is an ambassador for the agricultural industry and how people can use consumer choice to impact the environment in a positive way.",
+    native: {
+      text: "United States",
```
Comment (MattIPv4) [inline src/ambassadors/core.ts]: Should this be flagged as domesticated?

File: src/ambassadors/core.ts [hunk 9]
```diff
@@ -960,6 +1104,10 @@ const ambassadors = {
       "Polly was rehomed from another owner.\n\nWe had hoped that Polly would grow to be larger so she could join our other chickens, but unfortunately she did not and so was retired from being an ambassador in November 2022 and now lives at Maya's Tiny House.",
     mission:
       "She is an ambassador for the agricultural industry and how people can use consumer choice to impact the environment in a positive way.",
+    native: {
+      text: "China",
```
Comment (MattIPv4) [inline src/ambassadors/core.ts]: Ditto?

**Review by MattIPv4 — COMMENTED** [review]
For the sake of getting this merged, a few thoughts on marking a few more as domesticated -- I think the 'native' nomenclature as the property name is fine for now, with us making it clear when species have been domesticated.

Would you also mind making sure this branch is up-to-date with master? I think you'll need to bump the version again.

File: src/ambassadors/core.ts (hunk 4 above)
Comment (itsjayway) [reply src/ambassadors/core.ts]: yes, thank you

their entry states 

> Domesticated chinchillas are thought to be of the C. lanigera

of which Snork and Moomin both are

**Review by MattIPv4 — COMMENTED** [review]
👀 Looks like you need to run the version command again -- otherwise, this lgtm

File: src/ambassadors/core.ts [hunk 10]
```diff
@@ -950,16 +1096,20 @@ const ambassadors = {
...
     arrival: "2022-01",
-    retired: "2023-06",
+    retired: null,
     iucn: {
       id: null,
       status: "NE",
     },
     enclosure: "critters",
     story:
-      "Our rubber ducky isopods were born in captivity and ethically sourced.\n\nUnfortunately, our colony of isopods didn't breed enough to sustain themselves, with the final few passing away in June 2023.",
```
Comment (MattIPv4) [inline src/ambassadors/core.ts]: Looks like you've had a bad merge/rebase somewhere? Apologies for not catching that before

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

File: src/ambassadors/core.ts [hunk 11]
```diff
@@ -950,16 +1096,20 @@ const ambassadors = {
     sex: null,
     birth: "2022",
     arrival: "2022-01",
-    retired: "2023-06",
+    retired: null,
```
Comment (itsjayway) [inline src/ambassadors/core.ts]:
```suggestion
    retired: "2023-06",
```

File: src/ambassadors/core.ts [hunk 12]
```diff
@@ -950,16 +1096,20 @@ const ambassadors = {
...
     iucn: {
       id: null,
       status: "NE",
     },
     enclosure: "critters",
     story:
-      "Our rubber ducky isopods were born in captivity and ethically sourced.\n\nUnfortunately, our colony of isopods didn't breed enough to sustain themselves, with the final few passing away in June 2023.",
+      "Our rubber ducky isopods were born in captivity and ethically sourced.",
```
Comment (itsjayway) [inline src/ambassadors/core.ts]:
```suggestion
      "Our rubber ducky isopods were born in captivity and ethically sourced.\n\nUnfortunately, our colony of isopods didn't breed enough to sustain themselves, with the final few passing away in June 2023.",
```

File: src/ambassadors/core.ts [hunk 13]
```diff
@@ -937,10 +1078,15 @@ const ambassadors = {
...
       "She is an ambassador for the agricultural industry and how people can use consumer choice to impact the environment in a positive way.",
+    native: {
+      text: "China (Domesticated)",
+      source: "https://en.wikipedia.org/wiki/Silkie",
+    },
     clips: [],
     homepage: null,
     plush: null,
   },
+
```
Comment (MattIPv4) [inline src/ambassadors/core.ts]:
```suggestion
```

**Review by MattIPv4 — APPROVED** [review]
LGTM, thank you for all your work on this 💚

**Pull request merged**